sdk/keyvault/Azure.Security.KeyVault.Certificates/src/ImportCertificateOptions.cs

```diff
@@ -79,6 +80,12 @@ public ImportCertificateOptions(string name, byte[] certificate)
         /// </summary>
         public IDictionary<string, string> Tags => LazyInitializer.EnsureInitialized(ref _tags);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the certificate chain preserves its original order.
+        /// The default value is false, which sets the leaf certificate at index 0.
```

Comment:
Would it be interesting to explain how it would look when set to true or would it be obvious based on the current description?

Reply:
I am using the description provided by the service team in the specifications. This is just an option that was added because some customers want to preserve the original order of the certificate chain. Therefore, it is expected that they have context of what the certificate chain entails.